src/server/svr_func.c

```diff
@@ -2248,60 +2276,102 @@ static int
 check_single_entity_res(enum lim_keytypes kt, char *ename,
 	attribute *patr,
 	resource  *newr,
-	resource  *oldr)
+	resource  *oldr,
+	int	  subjobs,
+	job	  *pjob)
 {
 	char *kstr;
 	void               *ctx;
 	svr_entlim_leaf_t *plf;
 	int		   rc;
-	attribute          tmpval;
+	int i;
+	attribute  tmpval = {0};
 
-	if ((patr->at_flags & ATR_VFLAG_SET) == 0)
-		return No_Limit;
+	ET_LIM_DBG("entered", __func__)
```

Comment:
fixed in commit rc11 https://github.com/PBSPro/pbspro/pull/807/commits/6316bca03e34a3c756ae37b7ec6c6dd2297f1eb2

I have removed the unnecessary "entered" logs,
Keeping the other logs as DEBUG4 logs are meant for flooding the logs. this will help us in debugging bugs arising at customer site without rebuilding